### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by michael-je**
Update README.md to better reflect the current state of the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,21 +63,13 @@ Simply build and run the container:
 8. Run the bot:
```
Comment (morgaesis) [inline README.md:63]: Can you update the link to the guide to invite the bot to the [official documentation](https://discordpy.readthedocs.io/en/stable/discord.html)?

**Review by morgaesis — APPROVED** [review]
LGTM :+1:

File: README.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # The Lone Dancer - Discord music bot
-This project is still in development. The goal is to create a simple music playing bot for our discord server. It should be able to connect to a voice channel, accept basic commands and play back audio from youtube.
+
+This is a working discord bot centered around streaming music from **YouTube** into a voice channel. It is in active development and recieves regular feature updates and bug fixes.
```
[the remark this answers is not in the document]
Comment (michael-je) [reply README.md:3]: good point, done.

File: README.md (hunk 1 above)
Comment (michael-je) [reply README.md:63]: yup, done :)

**Review by thorduragust — APPROVED** [review]

**Pull request merged**